docs/source/guide/guide.myst

```diff
@@ -7,6 +7,7 @@ maxdepth: 2
 guide-getting-started.rst
 guide-executors.rst
 guide-observables.myst
+guide-guide-core-concepts.myst
```

Comment:
```suggestion
guide-core-concepts.myst
```
I guess this is preventing the new sections to be visible in the html RTD preview.

Reply:
good catch.